tla/etcdraft.tla

```diff
@@ -133,18 +135,21 @@ leaderVars == <<matchIndex, pendingConfChangeIndex>>
 VARIABLE 
     config
 VARIABLE 
-    reconfigCount
+    appliedConfChange
 
-configVars == <<config, reconfigCount>>
+configVars == <<config, appliedConfChange>>
 
 VARIABLE 
     durableState
+VARIABLE  
+    ready
```

Comment:
Missing type annotation. Could you also add an explanation here?

Reply:
I tried to add more comments to new states and actions. Please let me know if I need to put more details here.

Reply:
Looks good to me!